assets/minecraft/shaders/program/raytracer.fsh

```diff
@@ -261,7 +261,7 @@ vec3 pathTrace(Ray ray, out float depth) {
         // We didn't hit anything
         depth = far;
         float sunFactor = smoothstep(0.9987, 0.999, dot(ray.direction, sunDir));
-        return pow(sunFactor * SUN_COLOR + (1 - sunFactor) * SKY_COLOR, vec3(GAMMA_CORRECTION)) * weight;
+        return sunFactor * pow(SUN_COLOR, vec3(GAMMA_CORRECTION)) + (1 - sunFactor) * pow(SKY_COLOR, vec3(GAMMA_CORRECTION));
```

Comment:
This should probably be done when creating SKY_COLOR and SUN_COLOR
`const vec3 SUN_COLOR = pow(<current skycolor value>, vec3(GAMMA_CORRECTION));`